src/PrestaShopBundle/Resources/views/Admin/Sell/Order/CreditSlip/Blocks/credit_slip_options.html.twig

```diff
@@ -0,0 +1,56 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% trans_default_domain "Admin.Orderscustomers.Feature" %}
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(creditSlipOptionsForm, {
+  'method': 'POST',
+  'action': path('admin_credit_slips_process_options')})
+}}
```

Comment:
Dont forget the `form_error(form)`
See https://github.com/PrestaShop/PrestaShop/issues/14103

Reply:
It seems form_error never renders anything in `options forms`, because of the way its configured. Currently errors might only come from configuration to formHandler. If i add any form constraint it is ignored in handler and then lost after redirect.. Currently we have no options forms with constraints, but i think this should be considered in future.

Reply:
@matks, also one thing I'm concerned. Is it okay that the slip_prefix has no validation at all, whatever is written in input, will be saved to the database. (This behavior came from legacy).

Reply:
4fc128d1978fc595fbe41dbeec4f98eef54a53cb this commit solves the form error rendering problem

Reply:
@zuk3975 About initial feedback: dont forget these forms are meant to be extended by module developers 😉 so we must include `form_rest()` and `form_errors()` everywhere, even if they are useless in the standard config, because configuration can be customized

For your question: no you're right we should not allow everything to be a slip_prefix 😐 
@marionf Do you have an idea of what we could allow for slip prefix ? It is used before a file name for credit slips if I'm not wrong.

I think we should allow classic alphabet, maybe numbers, but not things like #*$°£ ?

Reply:
Marion already replied here https://github.com/PrestaShop/PrestaShop/pull/13903#discussion_r297658378 :smile:
However i have applied the regex from legacy validation isFileName. It allows alphanumeric and `_-.`. That seems fair for file name prefix

Reply:
@matks, okay I am a bit lost with all that form_rest and errors thing, so tried to play around with errors and form_rest. There the issue I've raised #14844